runner/src/internal.rs

```diff
@@ -293,6 +295,41 @@ impl RustBinaryOptions for BuildServer {
     }
 }
 
+#[derive(StructOpt, Clone)]
+pub struct RunFuzzTargets {
+    #[structopt(
+        long,
+        help = "name of a specific crate with fuzz-target. If not specified, runs all fuzz targets for all crates."
+    )]
+    pub crate_name: Option<String>,
+    #[structopt(
+        long,
+        help = "name of a specific fuzz-target. If not specified, runs all fuzz targets.",
+        requires("crate-name")
+    )]
+    pub target_name: Option<String>,
+    /// Additional `libFuzzer` arguments passed through to the binary
+    #[structopt(last(true))]
+    pub args: Vec<String>,
+}
+
+/// Partial representation of Cargo manifest files.
+///
+/// Only the fields that are required for extracting specific information (e.g., names of fuzz
+/// targets) are included.
+#[derive(serde::Deserialize, Debug)]
+pub struct CargoManifest {
+    #[serde(default)]
+    pub bin: Vec<CargoBinary>,
+}
+
+/// Partial information about a Cargo binary, as included in a Cargo manifest.
+#[derive(serde::Deserialize, Debug)]
+pub struct CargoBinary {
+    #[serde(default)]
+    pub name: String,
+}
```

Comment:
Would https://crates.io/crates/cargo_metadata work here? (we already use it elsewhere: https://github.com/project-oak/oak/blob/9f0ac739e47c6b713b3fd2838ad0ce4738192761/sdk/rust/oak_tests/src/lib.rs#L50-L53)

Reply:
This looks interesting, but I could not find any structs in `cargo_metadata` that matches the structure of a Cargo toml file. I particularly tried `Metadata`, and `Package`.